### PR TITLE
Add chat_group actions. Add tests.

### PR DESCRIPTION
**Pull request by reiterl**
https://github.com/OpenSlides/OpenSlides/pull/6111 is needed for this.

### CODE REVIEW

File: openslides_backend/models/models.py [hunk 1]
```diff
@@ -239,6 +239,7 @@ class Meeting(Model):
     jitsi_domain = fields.CharField()
     jitsi_room_name = fields.CharField()
     jitsi_room_password = fields.CharField()
+    enable_chat = fields.BooleanField()
```
Comment (FinnStutzenstein) [inline openslides_backend/models/models.py:242]: Please add this field to meeting.update (group f, next to the jitsi fields)

**Review by FinnStutzenstein — APPROVED** [review]
Code looks good!

File: openslides_backend/action/actions/chat_group/mixins.py [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+    def update_instance(self, instance: Dict[str, Any]) -> Dict[str, Any]:
+        chat_group = self.datastore.get(
+            FullQualifiedId(self.model.collection, instance["id"]), ["meeting_id"]
+        )
+        meeting = self.datastore.get(
+            FullQualifiedId(Collection("meeting"), chat_group["meeting_id"]),
+            ["enable_chat"],
+        )
+        if not meeting.get("enable_chat"):
+            raise ActionException("Chat is not enabled.")
```
Comment (jsangmeister) [inline openslides_backend/action/actions/chat_group/mixins.py:18]: You can extract this into its own function/mixin so you can use it in the `create` and `sort` as well.

File: tests/system/action/chat_group/test_sort.py [hunk 3]
```diff
@@ -0,0 +1,120 @@
...
+            }
+        )
+        response = self.request(
+            "chat_group.sort",
+            {"meeting_id": 222, "chat_group_ids": [32, 31]},
+        )
+        self.assert_status_code(response, 400)
+        assert "Id 32 not in db_instances." in response.json["message"]
+
+    def test_sort_another_section_db(self) -> None:
```
Comment (jsangmeister) [inline tests/system/action/chat_group/test_sort.py:82]: name is not fitting

File: tests/system/action/chat_group/test_update.py [hunk 4]
```diff
@@ -0,0 +1,58 @@
+from typing import Any, Dict
+
+from openslides_backend.permissions.permissions import Permissions
+from tests.system.action.base import BaseActionTestCase
+
+
+class ChatGroupDelete(BaseActionTestCase):
```
Comment (jsangmeister) [inline tests/system/action/chat_group/test_update.py:7]: name is not fitting

**Review by jsangmeister — CHANGES_REQUESTED** [review]
Please also add one test in either create or update where you try to assign a group from another meeting, which should fail

File: tests/system/action/chat_group/test_update.py (hunk 4 above)
Comment (reiterl) [reply tests/system/action/chat_group/test_update.py:7]: Renamed.

File: tests/system/action/chat_group/test_sort.py (hunk 3 above)
Comment (reiterl) [reply tests/system/action/chat_group/test_sort.py:82]: Renamed into: `test_sort_additional_chat_groups_in_meeting`

File: openslides_backend/action/actions/chat_group/mixins.py (hunk 2 above)
Comment (reiterl) [reply openslides_backend/action/actions/chat_group/mixins.py:18]: Done.

**Comment by reiterl** [comment]
Add group from different meeting tests to create and update.

File: openslides_backend/models/models.py (hunk 1 above)
Comment (reiterl) [reply openslides_backend/models/models.py:242]: Done. Added field enable_chat to meeting.update in group f.

**Review by jsangmeister — APPROVED** [review]

**Pull request merged**